### PR TITLE
Add Warner Bros. Discovery Service

### PR DESCRIPTION
**Pull request by eduardoaddad**
Add HBO Max, Discover+, CNN GO and TNT GO

### CODE REVIEW

File: internal/filtering/blocked.go [hunk 1]
```diff
@@ -379,6 +379,20 @@ var servicesData = []svc{{
...
+		"||hbomax.com^",
+		"||hbonow.com^",
+		"||hbonow.com^",
+		"||hbomax-images.warnermediacdn.com^",
+		"||art-gallery-latam.api.hbo.com^",
+		"||artist.api.cdn.hbo.com^",
+		"||hbogoasia.id^",
+		"||tntgo.tv^",
+		"||discoveryplus.com^",
+		"||go.cnn.com^",
```
Comment (ainar-g) [inline internal/filtering/blocked.go:394]: Some older rule sets aren't alphabetically sorted, unfortunately, but we strive to keep rules sorted.

File: client/src/helpers/constants.js [hunk 2]
```diff
@@ -347,6 +347,10 @@ export const SERVICES = [
         id: 'youtube',
         name: 'YouTube',
     },
+    {
+        id: 'warnerbrosdiscovery',
+        name: 'Warner Bros. Discovery',
+    },
```
Comment (ainar-g) [inline client/src/helpers/constants.js:353]: `client/src/components/ui/Icons.js` should be changed as well.

**Review by ainar-g — CHANGES_REQUESTED** [review]
Thanks for the contribution!  Here are some comments:

File: internal/filtering/blocked.go (hunk 1 above)
Comment (eduardoaddad) [reply internal/filtering/blocked.go]: done

**Review by ainar-g — CHANGES_REQUESTED** [review]
See the comment about icons.

**Comment by eduardoaddad** [comment]
> See the comment about icons.

How do I add icons?

**Comment by ainar-g** [comment]
Apologies for the long wait.  We've moved to using the same list of blockable services that AdGuard DNS uses, https://github.com/AdguardTeam/HostlistsRegistry.  Please open a PR there, and once accepted, it will be loaded to the next AdGuard Home release as well as in AdGuard DNS.  Again, thanks for the contribution and apologies for not reacting quicker.

**Pull request closed without merging**